tests/test_multisend.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from ape_safe.exceptions import SafeLogicError
+
+
+def test_asset(vault, token):
+    assert vault.asset() == token
+
+
+def test_default_operation(safe, token, vault, multisend):
+    amount = token.balanceOf(safe)
+    multisend.add(token.approve, vault, safe)
+    multisend.add(vault.transfer, safe, amount)
```

Comment:
not a comment on the test (the test is good), just typically a 4626 vault would do a `deposit` action to take in tokens, a `transfer` would be moving the shares (after processing the deposit)

Reply:
also, would think it'd be `token.approve(vault, amount)` then `vault.deposit(amount)`